.github/crowdin.yml

```
@@ -0,0 +1,10 @@
+project_id_env: CROWDIN_PROJECT_ID
+api_token_env: CROWDIN_PERSONAL_TOKEN
+base_path: "."
+preserve_hierarchy: true
+
+files:
+  - source: ".github/translations/*.po"
+    translation: "emhttp/languages/%locale%/*.po"
+  - source: "emhttp/languages/en_US/*.txt"
+    translation: "emhttp/languages/%locale%/*.txt" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)